src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -92,8 +122,10 @@ function processElement(cellElement) {
 			if (lifeLoss >= critLifeThreshold || npcName == 'Undaron') {
 				$(npcFastAttackElement).css('color', '#E7971F');
 			}
-			$(npcFastAttackElement).append(' (-' + lifeLoss + ' LP)');
+		//	$(npcFastAttackElement).append(' (-' + lifeLoss + ' LP)');
```

Comment:
u can just remove it instead of commenting out

Reply:
removed